.github/workflows/3rd-party-tests.yml

```diff
@@ -0,0 +1,89 @@
+name: Test on 3rd party libraries
+
+on:
+  push:
+    branches:
+      - "master"
+      - "sqlalchemy_pipeline"
```

Comment:
this branch can be removed (or not)

Reply:
I don't see this branch in your repo. Should it be removed?

Reply:
it's definetly there, I'm using it to run the pipeline in my fork https://github.com/caselit/psycopg/tree/sqlalchemy_pipeline

My comment was just to mention that I can remove before the merge